package/Codegen/Util.cs

```diff
@@ -540,6 +586,63 @@ public static IEnumerable<Command> GetCommands(string typeName)
             }
             return new Command[] { };
         }
-    }    
+
+        public static HashSet<MrType> eventArgsTypes = new (new NameEqualityComparer());
+        public static void FoundEventArgsType(MrType type)
+        {
+            if (type.GetFullName() == "System.Object")
+            {
+                return;
+            }
+            eventArgsTypes.Add(type);
+        }
+
+        public static Dictionary<string, IEnumerable<string>> eventArgsMethods { get; private set; } = new ();
+        public static IEnumerable<string> GetEventArgsMethods(string typeName)
+        {
+            if (Util.eventArgsMethods.TryGetValue(typeName, out var ms))
+            {
+                return ms;
+            }
+            return new string[] { };
+        }
+
+        private static string GetEventArgMethodParamType(string name, MrType paramType)
+        {
+            if (Util.DerivesFrom(paramType, $"{XamlNames.XamlNamespace}.DependencyObject")) {
+                return "tag: number";
+            }
+            return $"{name}: {GetTypeScriptType(paramType)}";
+        }
+
+        public static string GetEventArgsMethodArgs(MrType argType, string methodName)
+        {
+            argType.GetMethodsAndConstructors(out var methods, out var ctors);

```

Comment:
nit: If you're not going to use the out var, use `_`

```suggestion
            argType.GetMethodsAndConstructors(out var methods, out _);
```